concrete/src/Cookie/ResponseCookieJar.php

```diff
@@ -20,6 +26,28 @@ class ResponseCookieJar
      */
     protected $clearedCookies = [];
 
+    /**
+     * Set the value of the "secure" flag when it is set to null.
+     *
+     * @return $this
+     */
+    public function setSecureDefault(bool $default): self
+    {
+        $this->secureDefault = $default;
+
+        return $this;
+    }
+
+    /**
+     * Get the value of the "secure" flag when it is set to null.
+     *
+     * @return $this
```

Comment:
```suggestion
     * @return bool
```

Reply:
Fixed by 2e3a14f488441d0f741dbf25e8ec10c01c5c07c2 (we don't need that PHPDoc return type: it's already declared in the method signature).